src/seedu/addressbook/data/person/Name.java

```diff
@@ -60,4 +60,10 @@ public int hashCode() {
         return fullName.hashCode();
     }
 
+	@Override
+	public String getPrintableString(Printable... printables) {
```

Comment:
but no~ getPrintableString () should have no parameter ^^ you can look at other PR request to learn how others do this exercise ^^